### ISSUE TITLE
Fill handle will copy cells with 'copyable: false' (like passwords)

### ISSUE THREAD

**Issue by ottosson** [handsontable/handsontable#4529]
### Description
I want disable copying of columns and put `copyable: false` on the column options. But I can still copy the cells by using the fill handle. I would like to make a cell completely unable to be copied. Like password cells, or images in my case.
I would like to be able to set fillHandle setting to false, but this only seems to work with entire table and not specific columns/cells.

### Steps to reproduce
1. Set colOptions to {data: 1, type: 'password', `copyable: false`} 
2. Use fill handle to drag the value on to another cell

### Demo

https://jsfiddle.net/1mgzzd5e/1/

### Your environment
* Handsontable version: 0.34.4
* Browser Name and version: Firefox 57, Chrome 60
* Operating System: MacOS Sierra

**Comment by AMBudnik** [comment]
Hi @ottosson 

Copying and using the fillhandle don't mean the same. However, as you rightly spotted the `fillHandle` option can be provided with the `false` value that disables the values to be populated.

You should be able to use the `cells` method like presented at https://jsfiddle.net/1t0qdzcj/ to pick cells that can block the `fillHandle` but it's not working. I am marking the topic as a bug report.

Thank you very much for sharing.

**Comment by ottosson** [comment]
Shouldn't it be possible to put `fillHandle` on column too, not just cells?

Anyway, how would you go by making a cell completely locked for copying, if possible?

Thanks!

**Comment by AMBudnik** [comment]
Yes, it should work in the `columns` as well. The code is working pretty similar to what we are doing for `cells`. 

You can use another way to accomplish this goal - the `beforeChange` hook. Here is a demo https://jsfiddle.net/97q4xk34/

I am returning `false` for the `beforeChange` hook, where the change is planned for the index of `2` (column `C`) and the change type is `Autofill.fill`.

**Comment by ottosson** [comment]
Nice, that will be alright for now.

Thanks a lot!

**Comment by AMBudnik** [comment]
You're welcome @ottosson 

and again thanks for sharing the issue. We weren't aware that we can set the `fillHandle` in lower levels of table structure.

**Comment by pnowak** [comment]
This issue is still replicable using v8.0.0. Here's an updated demo - https://jsfiddle.net/s8fjt3wv/

**Comment by budnix** [comment]
The `copyable` option is tied to the copy/paste process itself (the CopyPaste plugin).

If you want to disable similar behavior for other features (like Autofill), I would recommend use `readOnly`. If you don’t want the cell styling to change, you can also set `readOnlyCellClassName: ''` https://jsfiddle.net/budnix/Lwveb9p2/1/
